src/models/EventMapper.php

```diff
@@ -453,6 +453,8 @@ public function transformResults($results, $verbose)
                             . $row['ID'] . '/comments/reported';
                         $list[$key]['reported_talk_comments_uri'] = $base . '/' . $version . '/events/'
                             . $row['ID'] . '/talk_comments/reported';
+                        $list[$key]['pending_claims_uri'] = $base . '/' . $version . '/events/'
```

Comment:
Is it possible to show this URI only to people that actually can do something with it? So AFAIK that'd be event-admins. I find it nicer to only show people stuff they actually can use. And most people will get a 403 (or 401) when calling that URI.